database/postgres/pg.test.js

```diff
@@ -35,22 +40,25 @@ describe('Test PG DB', ()=>{
       it('it should execute all queries if required arguments are passed into createPgAccount', async ()=>{
         await createPgAccount('username', 'password')
         expect(mockClient.query).toHaveBeenCalledTimes(3)
-        expect(mockClient.query).toHaveBeenNthCalledWith(1, `CREATE DATABASE IF NOT EXISTS username`)
-        expect(mockClient.query).toHaveBeenNthCalledWith(2, `CREATE USER IF NOT EXISTS username WITH ENCRYPTED password 'password'`)
-        expect(mockClient.query).toHaveBeenNthCalledWith(3, `GRANT ALL PRIVILEGES ON DATABASE username TO username`)
+        expect(mockClient.query.mock.calls[0][0]).toEqual(`CREATE DATABASE IF NOT EXISTS $1`)
+        expect(mockClient.query.mock.calls[0][1][0]).toEqual('username')
```

Comment:
Instead of `[0][1][0]`
Could you use `expect(mockClient.query.mock.calls[0][1]).toEqual(['username'])`
So it looks cleaner